oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/rdb/RDBDocumentNodeStoreBuilder.java

```diff
@@ -154,6 +157,18 @@ public RDBDocumentNodeStoreBuilder setFullGCExcludePaths(@NotNull String[] exclu
         return thisBuilder();
     }
 
+    @Override
+    public RDBDocumentNodeStoreBuilder setFullGCMode(int v) {
+        // fullGC modes are not supported for RDB
+        log.warn("FullGC modes are not supported for RDB");
```

Comment:
this should return the `builder` object.

Reply:
Fixed.